rstcheck.py

```diff
@@ -303,7 +268,7 @@ def run_in_subprocess(code, filename_suffix, arguments):
     """Return None on success."""
     temporary_file = tempfile.NamedTemporaryFile(mode='w',
                                                  suffix=filename_suffix)
-    temporary_file.write(code)
+    temporary_file.write(code.encode('utf-8'))
```

Comment:
python 2 does not understand ASCII symbols

./rstcheck.py examples/ascii.rst

```python
  File "rstcheck.py", line 271, in run_in_subprocess
    temporary_file.write(code)  # .encode('utf-8'))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)
```

I fixed it there https://github.com/myint/rstcheck/commit/01542b09cced0bbe4a72fd5c8e56246043c67cfd#diff-343c5bc7549fdcfa7acef23ff2cba6b8R271